docs/freqai.md

```diff
@@ -0,0 +1,265 @@
+
+## Configuring the bot
+### Example config file
+The user interface is isolated to the typical config file. A typical Freqai
+config setup includes:
+
+```json
+    "freqai": {
+                "timeframes" : ["5m","15m","4h"],
+                "full_timerange" : "20211220-20220220",
+                "train_period" : "month",
+                "backtest_period" : "week",
+                "identifier" :  "unique-id",
+                "base_features": [
+                        "rsi",
+                        "mfi",
+                        "roc",
+                ],
+                "corr_pairlist": [
+                        "ETH/USD",
+                        "LINK/USD",
+                        "BNB/USD"
+                ],
+                "train_params" : {
+                        "period": 24,
+                        "shift": 2,
+                        "drop_features": false,
+                        "DI_threshold": 1,
+                        "weight_factor":  0,
+                },
+                "SPLIT_PARAMS" : {
+                    "test_size": 0.25,
+                    "random_state": 42
+                },
+                "CLASSIFIER_PARAMS" : {
+                    "n_estimators": 100,
+                    "random_state": 42,
+                    "learning_rate": 0.02,
+                    "task_type": "CPU",
+                },
+        },
+
+```
+
+### Building the feature set
+
+Most of these parameters are controlling the feature data set. The `base_features`
+indicates the basic indicators the user wishes to include in the feature set.
+The `timeframes` are the timeframes of each base_feature that the user wishes to
+include in the feature set. In the present case, the user is asking for the
+`5m`, `15m`, and `4h` timeframes of the `rsi`, `mfi`, `roc`, etc. to be included
+in the feature set.
+
+In addition, the user can ask for each of these features to be included from
+informative pairs using the `corr_pairlist`. This means that the present feature
+set will include all the `base_features` on all the `timeframes` for each of
+`ETH/USD`, `LINK/USD`, and `BNB/USD`.
+
+`shift` is another user controlled parameter which indicates the number of previous
+candles to include in the present feature set. In other words, `shift: 2`, tells
+Freqai to include the the past 2 candles for each of the features included
+in the dataset.
+
+In total, the number of features the present user has created is:_
+
+no. `timeframes` * no. `base_features` * no. `corr_pairlist` * no. `shift`_
+3 * 3 * 3 * 2 = 54._
+
+### Deciding the sliding training window and backtesting duration
+
+`full_timerange` lets the user set the full backtesting range to train and
+backtest through. Meanwhile `train_period` is the sliding training window and
+`backtest_period` is the sliding backtesting window. In the present example,
+the user is asking Freqai to train and backtest the range of `20211220-20220220` (`month`).
```

Comment:
Period parameters are now integers instead of strings. Needs to be reflected in the doc.